qatrack/qa/signals.py

```
@@ -287,3 +292,33 @@ def check_approved_statuses(*args, **kwargs):
+
+
+@receiver(post_save, sender=models.UnitTestCollection, dispatch_uid="update_active_unit_test_collections_utc_post_save")
+@receiver(post_delete, sender=models.UnitTestCollection, dispatch_uid="update_active_unit_test_collections_utc_post_delete")
+def update_active_unit_test_collections_for_unit_utc(*args, **kwargs):
+    unit = kwargs['instance'].unit
+    set_active_unit_test_collections_for_unit_cache(unit)
+
+
+@receiver(post_save, sender=Unit, dispatch_uid="update_active_unit_test_collections_unit_post_save")
+@receiver(post_delete, sender=Unit, dispatch_uid="update_active_unit_test_collections_unit_post_delete")
+def update_active_unit_test_collections_for_unit(*args, **kwargs):
```

Ruff (0.15.17)

[warning] Missing type annotation for `*args`

(ANN002)